report.html

```diff
@@ -0,0 +1,3702 @@
+{
```

Comment:
are we always planning to check in this file?

Reply:
Nope, deleted.

Reply:
can you add to the gitignore file